app/apps/open_zaak/mocking_views.py

```diff
@@ -3,8 +3,8 @@
 from apps.open_zaak.case_type.services import CaseTypeService
 from apps.open_zaak.catalog.services import CatalogService
 from apps.open_zaak.information_object_type.services import InformationObjectTypeService
-from apps.open_zaak.state.services import StateService
-from apps.open_zaak.state_type.services import StateTypeService
+from apps.open_zaak.state.services import LegacyStateService
+from apps.open_zaak.state_type.services import LegacyStateTypeService
```

Comment:
De State in de module zijn wrappers om status in open-zaak, dus die zou ik niet per se legacy noemen. 
Alleen de stadia die uit Top/BWV komen zijn legacy states neem ik aan. Ik vraag me af of dit ook hernoemd moet worden.

Overigens wordt niks onder de open_zaak module gebruikt. Misschien dat het straks weer relevant wordt als er echt een instance bij Basis geïnstalleerd wordt, en we voor de 'eventual consistency' gaan :)

Reply:
Fixed